packages/request-client.js/test/index.test.ts

```diff
@@ -1271,6 +1278,7 @@ describe('index', () => {
       });
 
       const request = await requestNetwork.createRequest({
+        disablePaymentDetection: true,
```

Comment:
why do we need to disable payment detection? is it tested somewhere else?

Reply:
Good question. The goal of this test if it works with a salt forced. The payment detection test is made later.